app/backend/Startup.cs

```diff
@@ -2,13 +2,18 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using System.IO;
 
 namespace backend
 {
     public class Startup
     {
         public Startup(IConfiguration configuration)
         {
+            var builder = new ConfigurationBuilder()
```

Comment:
This never got used but would be how the settings are access for passing variables and environment variables